identity-android/src/main/java/com/android/identity/android/mdoc/deviceretrieval/DeviceRetrievalHelper.java

```diff
@@ -38,7 +38,6 @@
 
 import java.security.KeyPair;
 import java.security.PublicKey;
-import java.util.Collections;
```

Comment:
This change doesn't seem germane to this PR and should probably be in a general cleanup instead.